### ISSUE TITLE
How do I get the last item of a collection to show as a summary on a blog?

### ISSUE THREAD

**Issue by syelan** [getgrav/grav#858]
Hi;

I have a page template that must pull its summary from the last published page in an (unpublished) blog section in another part of the website.

I've gotten to the point were I've been able to print all fo the summaries of the pages below the section *seasons* , but I can't find a way to print just the last one. Here is the code I have working in my template:

```
		<div class="header">
		  <div class="header_content">
		    <h1>{{ page.header.title }}</h1>

		    {% for p in page.find('/seasons').children if p != page %}
		    	<p class="lead">{{p.route}}—{{ p.summary|striptags}}</p>
		    {% endfor %}

		  </div>
		</div>
```
This is the structure of the site:
```
_
 |
 — contents
 |
 — seasons
    |
    — Page 1
    |
    — Page 2
    |
    — Page 3
```
What I want is to display just Page 3 as a summary in another page (which is a blog of the Contents section), but I get all Pages 1, 2 and 3.

I'm learning GravCMS, but even though the documentation is great, I can't find a way to use `isLast()` properly. In my case, I'm using 'date' to sort the last one.

Can anyone help with this please?

**Comment by rhukster** [comment]
According to Twig docs (http://twig.sensiolabs.org/doc/filters/last.html) you should be able to do this:

```
        <div class="header">
          <div class="header_content">
            <h1>{{ page.header.title }}</h1>

            {% for p in page.find('/seasons').children|last if p != page %}
                <p class="lead">{{p.route}}—{{ p.summary|striptags}}</p>
            {% endfor %}

          </div>
        </div>
```

**Comment by syelan** [comment]
Hi rhukster! Thank you for your reply! I tried your suggestion
```
            {% for p in page.find('/seasons').children|last if p != page %}
                <p class="lead">{{p.route}}—{{ p.summary|striptags}}</p>
           {% endfor %}
```
and for some reason that returned empty, so I clicked the link you suggested, and that reminded me about  [this in the Grav cookbook](https://learn.getgrav.org/cookbook/twig-recipes). Following that I changed it to 
```
		    {% for p in page.find('/seasons').children.order('date', 'desc').slice(0, 1) %}
```
and that did it. It's now correctly displaying the Page with the latest date as a summary in my Contents blog. Thank you for the suggestions!
